src/gt4py/next/program_processors/codegens/gtfn/gtfn_ir.py

```diff
@@ -96,15 +96,59 @@ class Backend(Node):
     domain: Union[SymRef, CartesianDomain, UnstructuredDomain]
 
 
+def _is_ref_or_tuple_expr_of_ref(expr: Expr) -> bool:
+    if (
+        isinstance(expr, FunCall)
+        and isinstance(expr.fun, SymRef)
+        and expr.fun.id == "tuple_get"
+        and len(expr.args) == 2
+        and _is_ref_or_tuple_expr_of_ref(expr.args[1])
+    ):
+        return True
```

Comment:
Is this case tested?

Reply:
This is indeed tested, the `_process_elements` creates such `gtfn_ir` expressions for example in the `test_multicopy` case.